via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -347,9 +348,12 @@ export default function Transcript({
     return filterTranscript(transcript.segments, filter);
   }, [filter, transcript]);
 
+  useOverflowAnchor(scrollRef, element => element.querySelectorAll('li'));
+
   return (
     <ScrollContainer borderless>
       <Scroll
+        style={{ overflowAnchor: 'none' }} // TODO Remove this!
```

Comment:
This is used just to test this on any browser. To be removed before this PR is ready.